packages/tasks/src/balances.ts

```diff
@@ -35,65 +35,70 @@ const balancesError = `
 const summarizeTokens = (tokens: any[]) => {
   let table = {} as any;
   tokens.forEach((token) => {
-    if (!table[token.chain_name]) {
-      table[token.chain_name] = {};
-    }
-    const balance = parseFloat(token.balance).toFixed(2);
-    if (parseFloat(token.balance) > 0) {
-      if (token.coin_type === "Gas") {
-        table[token.chain_name].gas = balance;
-      } else if (token.symbol === "ZETA") {
-        table[token.chain_name].zeta = balance;
-      } else if (token.coin_type === "ERC20") {
-        table[token.chain_name].erc20 =
-          (table[token.chain_name].erc20
-            ? table[token.chain_name].erc20 + " "
-            : "") +
-          balance +
-          " " +
-          token.symbol;
-      } else if (token.coin_type === "ZRC20") {
-        table[token.chain_name].zrc20 =
-          (table[token.chain_name].zrc20
-            ? table[token.chain_name].zrc20 + " "
-            : "") +
-          balance +
-          " " +
-          token.symbol;
+    if (token && token.chain_name) {
+      if (!table[token.chain_name]) {
+        table[token.chain_name] = {};
+      }
+      const balance = parseFloat(token.balance).toFixed(2);
+      if (parseFloat(token.balance) > 0) {
+        if (token.coin_type === "Gas") {
+          table[token.chain_name].gas = balance;
+        } else if (token.symbol === "ZETA") {
+          table[token.chain_name].zeta = balance;
+        } else if (token.coin_type === "ERC20") {
+          table[token.chain_name].erc20 =
+            (table[token.chain_name].erc20
+              ? table[token.chain_name].erc20 + " "
+              : "") +
+            balance +
+            " " +
+            token.symbol;
+        } else if (token.coin_type === "ZRC20") {
+          table[token.chain_name].zrc20 =
+            (table[token.chain_name].zrc20
+              ? table[token.chain_name].zrc20 + " "
+              : "") +
+            balance +
+            " " +
+            token.symbol;
+        }
       }
     }
   });
   return table;
 };
 
 const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
+  const client = new ZetaChainClient({
+    network: args.mainnet ? "mainnet" : "testnet",
+  });
   const spinner = ora("Fetching balances...");
   if (!args.json) {
     spinner.start();
   }
   const { ethers, config } = hre as any;
   const pk = process.env.PRIVATE_KEY;
-  let address: string;
-  let btc_address: any;
+  let evm: string;
```

Comment:
Can we go back to `evmAddress` and `btcAddress`?

Reply:
https://github.com/zeta-chain/toolkit/commit/1c75e0d8148eb8b0c06844cf293b64016bc570e8